app/components/forms/login-form.js

```diff
@@ -45,6 +49,10 @@ export default class LoginForm extends Component.extend(FormMixin) {
 
   @action
   async submit(e) {
+    this.set('counter', this.counter + 1);
+    if (this.counter > 1) {
+      this.set('captcha', true);
+    }
```

Comment:
Only increment on auth failure and instead of keeping 2 variables, just create a computed property based on count

Reply:
ok